lib/ddtrace/contrib/rack/request_queue.rb

```diff
@@ -0,0 +1,33 @@
+module Datadog
+  module Contrib
+    module Rack
+      # QueueTime simply...
+      module QueueTime
+        REQUEST_START = 'HTTP_X_REQUEST_START'.freeze
+
+        module_function
+
+        def get_request_start(env, now = Time.now.utc)
+          header = env[REQUEST_START]
+
+          if header
+            # nginx header is in the format "t=1512379167.574"
```

Comment:
for the current testing phase, only nginx is required; I'll iterate over other formats if they're different.